component/component/statefuleset-resize-controller.jsonnet

```diff
@@ -103,7 +103,7 @@ local resizeClusterRoleBinding = kube.ClusterRoleBinding('appcat:job:resizejob')
 };
 
 // Curently we only need this for redis.
-if params.services.vshn.enabled && (params.services.vshn.redis.enabled || params.services.vshn.services.mariadb.enabled) then {
+if params.services.vshn.enabled && (params.services.vshn.redis.enabled || params.services.vshn.mariadb.enabled) then {
```

Comment:
should keycloak be added here as well?

Reply:
There are no PVCs attached to Keycloak, sothere's nothing to resize.